src/openrct2/entity/MoneyEffect.cpp

```diff
@@ -172,6 +173,18 @@ void MoneyEffect::Paint(PaintSession& session, int32_t imageDirection) const
 {
     PROFILED_FUNCTION();
 
+    if (gScreenFlags & SCREEN_FLAGS_TITLE_DEMO)
+    {
+        // Don't render any money in the title screen.
+        return;
+    }
```

Comment:
I believe there is some code that removes guest purchases when the title sequence player loads a park. With this bit of code, that part can likely be removed.

Reply:
Guest purchases are no longer conditional, this has to stay or it will show guests purchasing stuff in the title screen.

Reply:
Yes, I talking about the part that removes the money effects when loading a park. I'll try to find this tonight when I'm home if you haven't found it them already.

Reply:
Sorry misunderstood that, yeah makes sense to remove that code that removes the money.